lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -209,6 +219,31 @@ defmodule ExUnit.Case do
 
       config :logger, backends: []
 
+  ## Tmp Dir
+
+  ExUnit automatically creates a temporary directory for tests tagged with
+  `:tmp_dir` and puts the path to that directory into the test context.
+  The directory is removed before being created to ensure we start with a blank
+  slate.
+
+  The temporary directory path is unique (includes the test module and test name)
+  and thus appropriate for running tests concurrently. You can customize it by
+  setting the tag to a string, e.g.: `tmp_dir: "my_path"`.
```

Comment:
It seems this option actually changes the path name but the name is still nested inside "module/test". Is this expected? I was thinking that, if you pass your own `tmp_dir`, then we should give it to the test as is... but I don't have a good use case for that. Maybe we should keep it simple and allow it to only be booleans for  now?

Reply:
Yes, I made it like this on purpose, I think keeping the test under tmp/:module/:name keeps things more tidy but yeah, it isn't obvious. My thinking was the actual path isn't that important, just the final directory is. I pointed out an existing test where setting custom path is useful, see the other comment.

Reply:
Good call. Let's just document it then this configures the directory inside the module/test folder.